java/client/src/org/openqa/selenium/net/UrlChecker.java

```diff
@@ -79,7 +79,7 @@ public void waitUntilAvailable(long timeout, TimeUnit unit, final URL... urls)
         while (true) {
           for (URL url : urls) {
             try {
-              log.fine("Polling " + url);
+              log.finest("Polling " + url);
```

Comment:
It is not clear why you've chosen `finest` over `fine`. Could you please add some explanation (possibly to this PR, rather than as a comment) about the reasoning behind this choice?

Reply:
There are several places where you've made this change. The comment above applies equally to all of them :)

Reply:
Sure. Here is the DOI table for selenium/client:

sequence | subject | low boundary inclusive | high boundary exclusive | log level
-- | -- | -- | -- | --
1557327791554 | client | 0 | 1.0653332 | FINEST
1557327791554 | client | 1.0653332 | 2.1306665 | FINER
1557327791554 | client | 2.1306665 | 3.1959996 | FINE
1557327791554 | client | 3.1959996 | 4.261333 | INFO
1557327791554 | client | 4.261333 | 5.3266664 | WARNING
1557327791554 | client | 5.3266664 | 6.3919992 | SEVERE

Log levels will be transformed to `FINEST` anytime their DOI values within [0, 1.0653332). This log had a DOI value of 0, most likely because it hasn't been interacted with in some time or not frequently enough. In fact, it seems that it was [2 years ago](https://github.com/SeleniumHQ/selenium/commit/715f0121fbe6ddcb75a18c473351a2eb855a07e3), which is probably greater than 1,000 commits ago (the threshold we used for this analysis).

We can go back further in time to see if it makes a difference (I am curious now).

Reply:
After setting N to 24,000, I am finding that the DOI for that log is still 0. I am wondering if we are losing it in some refactoring of some kind or that there's just so much history in selenium that the DOI is decaying due to other changes. We'll investigate. Filed https://github.com/ponder-lab/Logging-Level-Evolution-Plugin/issues/217.